### DISCUSSION TITLE
adding indexes - > cant refresh stache on live server

### DISCUSSION THREAD

**Discussion by HRR1337** [statamic/cms#5168] [Troubleshooting] [answered]
Hi there,

I work on a windows pc, my live server is ubuntu. 

Adding indexes to my stache config 

```
 'indexes' => [
        'status',
        'redirect',
        'featured_event',
        'item_is_inactief',
    ],
```

gets the following error when I want to refresh the stache on the live server (on my local dev env its working).

Clearing Stache...
Warming Stache...

```
   Error

  Call to undefined method Statamic\Entries\Collection::get()

  at vendor/statamic/cms/src/Query/ResolveValue.php:43
     39▕         if (method_exists($item, 'value')) {
     40▕             return $item->value($name);
     41▕         }
     42▕
  ➜  43▕         return $item->get($name);
     44▕     }
     45▕ }
     46▕

      +3 vendor frames
  4   [internal]:0
      Statamic\Stache\Indexes\Value::Statamic\Stache\Indexes\{closure}()

      +27 vendor frames
  32  please:37
      Illuminate\Foundation\Console\Kernel::handle()
```

How can I solve this issue?

p.s. using the latest statamic version.

**Comment by ryanmitchell** [comment, marked as answer]
This was answered over on Discord. The indexes were stored globally instead of scoped to the collection.

**Reply by jasonvarga** [reply]
(In the config file, add `indexes` to the `entries` store, instead of just at the top level.)
